cli/Cargo.toml

```diff
@@ -5,7 +5,9 @@ authors = ["Eric Freitag <eric@phylum.io>"]
 edition = "2018"
 
 [features]
+default = ["selfmanage"]
 phylum-online = []
+selfmanage = ["dirs"]
```

Comment:
> There is an implicitly added dirs feature here that is useless. But with Rust 1.60, we can use the new dep: syntax to avoid this. See details in The Cargo Book

I'm aware of this new feature, however I'm not sure about the MSRV guarantees of `phylum`. Is there any guideline wrt this?